docs/pages/es/project/CONTRIBUTING-gitflow.md

```diff
@@ -0,0 +1,95 @@
+# Trabajando por Fork
+Simplemente dirígete a la página de GitHub y haz clic en el botón "Fork". Es así de simple. Una vez que hayas hecho eso, puedes usar tu cliente git favorito para clonar tu repositorio o simplemente dirigirse directamente a la línea de comando:
+
+## Clona tu fork en tu máquina local
+```
+git clone git@github.com:USERNAME/FORKED-PROJECT.git
+```
+Mantener Tu Fork Actualizado
+Si bien este no es un paso necesario, si planeas hacer algo más que una pequeña solución rápida, querrás asegurarte de mantener tu fork actualizado rastreando el repositorio "upstream" original que bifurcaste. Para hacer esto, deberás agregar un "remote":
+
+## Agregar repositorio 'upstream' a la lista de "remotes"
+```
+git remote add upstream https://github.com/layer5io/meshery.git
+```
+("meshery" se usa como repositorio de ejemplo. Asegúrate de hacer referencia al repositorio _actual_ al que está contribuyendo, por ejemplo, "meshery-linkerd").
+
+## Verifica el nuevo "remote" llamado 'upstream'
+```
+git remote -v
+```
+Siempre que quieras actualizar tu fork con los últimos cambios de 'upstream', primero deberás buscar las ramas del repositorio 'upstream' y los últimos "commits" para llevarlos a tu repositorio:
+
+## Obtener desde el "remote" 'upstream'
+```
+git fetch upstream
+```
+
+## Ver todas las ramas, incluidas las de 'upstream'
+```
+git branch -va
+```
+Ahora, revisa tu rama "master" y combina la rama "master" del repositorio 'upstream':
+
+## Haz "checkout" a tu rama "master" y haz "merge" de 'upstream'
+```
+git checkout master
+git merge upstream/master
+```
+Si no hay "commits" únicos en la rama "master" local, git simplemente realizará un avance rápido. Sin embargo, si has realizado cambios en "master" (en la gran mayoría de los casos, probablemente no deberías hacerlo; consulta la siguiente sección, es posible que tengas que lidiar con conflictos. Al hacerlo, ten cuidado de respetar los cambios realizados anteriormente.
+
+Ahora, tu rama "master" local está actualizada con todo lo modificado en 'upstream'.
+
+**Crear una "Branch"** (haciendo tu trabajo)
+Siempre que comiences a trabajar en una nueva función o corrección de "bugs", es importante que crees una nueva rama. No solo es un flujo de trabajo de git adecuado, sino que también mantiene tus cambios organizados y separados de la rama "master" para que puedas enviar y administrar fácilmente múltiples "pull requests" para cada tarea que completes.
```

Comment:
Don't translate "fast-forward". So change "realizará un avance rápido" to "realizara un fast-forward".

finish this paragraph with a closing )